<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptiVision</class>
 <widget class="QMainWindow" name="OptiVision">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OptiVision</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(235, 255, 251);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Welcome_Label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text Semibold</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Register:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Email_Label">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>211</y>
      <width>56</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Password_Label">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>294</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EmailEditBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>251</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background: rgb(255, 255, 255);
	border-radius:10px;
	border:1px solid
}
QLineEdit:focus{
	background: rgb(255, 255, 255);
	border-radius:10px;
	border:1px solid #aa4f47
}
</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password1EditBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background: rgb(255, 255, 255);
	border-radius:10px;
	border:1px solid
}
QLineEdit:focus{
	background: rgb(255, 255, 255);
	border-radius:10px;
	border:1px solid #aa4f47
}
</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="OptiVision_Image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>221</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 255, 225);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Colored.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="RegisterButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>500</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	background-color: rgb(246, 255, 225);
	border-radius:5px;
	border: 1px solid
}
QPushButton:hover {	
	background-color: rgb(237, 248, 255);
}
</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>RegisterIcon.png</normaloff>RegisterIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>251</width>
      <height>781</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 255, 225);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Name_Label">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>130</y>
      <width>56</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>129</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameEditBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background: rgb(255, 255, 255);
	border-radius:10px;
	border:1px solid
}
QLineEdit:focus{
	background: rgb(255, 255, 255);
	border-radius:10px;
	border:1px solid #aa4f47
}
</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ConfirmPassword_Label">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>390</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>390</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password2EditBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>430</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background: rgb(255, 255, 255);
	border-radius:10px;
	border:1px solid
}
QLineEdit:focus{
	background: rgb(255, 255, 255);
	border-radius:10px;
	border:1px solid #aa4f47
}
</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>560</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{	
	background-color: rgb(235, 255, 251);
	border-radius:5px;
}
QPushButton:hover {	
	background-color: rgb(246, 255, 225);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>BackIcon.png</normaloff>BackIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ErrorMessage_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>550</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>Welcome_Label</zorder>
   <zorder>Email_Label</zorder>
   <zorder>Password_Label</zorder>
   <zorder>EmailEditBox</zorder>
   <zorder>Password1EditBox</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>RegisterButton</zorder>
   <zorder>label_9</zorder>
   <zorder>OptiVision_Image</zorder>
   <zorder>Name_Label</zorder>
   <zorder>label_10</zorder>
   <zorder>NameEditBox</zorder>
   <zorder>ConfirmPassword_Label</zorder>
   <zorder>label_12</zorder>
   <zorder>Password2EditBox</zorder>
   <zorder>backButton</zorder>
   <zorder>ErrorMessage_label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
